Detectors/GlobalTracking/include/GlobalTracking/MatchTOF.h

```diff
@@ -84,7 +85,7 @@ class MatchTOF
   void init();
 
   ///< perform all initializations
-  void initWorkflow(const std::vector<o2::dataformats::TrackTPCITS>* trackArray, const std::vector<Cluster>* clusterArray, const o2::dataformats::MCTruthContainer<o2::MCCompLabel>* toflab, const std::vector<o2::MCCompLabel>* itslab, const std::vector<o2::MCCompLabel>* tpclab);
+  void initWorkflow(const gsl::span<const o2::dataformats::TrackTPCITS>* trackArray, const gsl::span<const Cluster>* clusterArray, const o2::dataformats::MCTruthContainer<o2::MCCompLabel>* toflab, const gsl::span<const o2::MCCompLabel>* itslab, const gsl::span<const o2::MCCompLabel>* tpclab);
```

Comment:
Use ``const gsl::span<const o2::dataformats::TrackTPCITS> trackArray`` (also reference can be used): the span itself is just a pair of pointer + size indicator. The same for all cases below.

Reply:
pointers removed